source/getting_started/how-to.rst

```diff
@@ -0,0 +1,105 @@
+
+    For Linux, use the ``dd`` tool to easily flash the image file to the USB disk. For example:
+
+    .. code-block:: bash
+
+        $ sudo dd if=cel_kbl_gptimage.img of=/dev/sdc bs=1M
+
+    .. caution::
+        You **MUST** change the */dev/sdc* to the real USB disk device.
+
+        Be sure all the buffered data is written to the USB disk before removing the USB disk (e.g. run the ``sync`` command in Linux).
+
+    For Windows, use the ``rufus`` tool to flash the image file to the USB disk.
+
+#. Boot from the USB disk to Android directly.
+
+    Plug in the USB disk and power on. Then, press a function key to enter the BIOS boot menu. Select **boot** from the USB disk. Now, you can boot from the USB disk to Android directly.
+
+    If you choose to save data to this Android device and you have not removed the USB disk, the data will be saved to the USB disk. You can then use the saved data on other computers.
+
+    .. note::
+        #. If you already installed Android in the internal storage, disable the internal storage if you are using USB live boot.
+        #. If you want to use one USB disk in two computers, make sure the lock status is same in each computer.
+
+How to dual boot |C| and Windows
+--------------------------------
+
+You may want to install Windows and |C| Anroid on one device, and select which OS to boot everytime the device is powering up. To achieve the dual boot, you need to install Windows and |C| on two storage devices seperately. There are two approaches:
+
+#. Use Kernel Flinger UEFI executables:
+
+    The Kernel Flinger UEFI installer does not prompt for the target storage device to install. You need to disconnect the Windows installed storage device first, connect the storage back to the device after installing |C|, and then select which OS to boot from the BIOS boot menu.
+
+#. Install using GRUB installer image:
+
+    A GRUB installer image is created if you built the image with the following command. The GRUB installer provides interactive prompt for you to select which storage device to install |C|.
+
+    .. code-block:: bash
+
+        $ make SPARSE_IMG=true project_celadon-efi -j $(nproc)
+
+    Substitute *$(nproc)* to the appropriate value according to the CPU cores on your build server.
+
+    After installing |C| Android and Windows, you can select which OS to boot from the BIOS boot menu. Or, you can add a boot item to the GRUB menu for Windows, and always boot into the GRUB bootloader and select which OS to boot from the GRUB menu.
+
+How to change the partition size in |C| image
+---------------------------------------------
+
+An Android system typically comprises multiple disk partitions, each parition is associated with an *\*.img* file in the |C| image. In addition, |C| also supports a whole GPT image to be flashed to the entire disk.
+Some partitions, such as the system partition, are huge, but some of them are empty, the **SPARSE** image format is used to reduce the image size.
```

Comment:
Suggested rewrite: 
Some partitions (e.g. the system partition) are huge, but some of them are empty. The **SPARSE** image format is used to reduce the image size.